app.py

```diff
@@ -14,6 +13,30 @@
 async def start(request):
     return text("pong")
 
+@app.route("/matrix", methods=["GET", "POST"])
```

Comment:
Let's call this route `/ranking`, and let `matrix` refer to a term document matrix.